src/main/dotnet/ProjectFileParser/Jsonify.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using Microsoft.Build.Construction;
+using Microsoft.Build.Evaluation;
+using Newtonsoft.Json.Linq;
+
+namespace ProjectFileParser
```

Comment:
The test included is targeting net framework 2 and 4.5
It would be cool if someone could try on mono